apps/sms/views/shared/js/selection_handler.js

```diff
@@ -72,9 +72,8 @@ SelectionHandler.prototype = {
       return;
     }
     var target = event.target;
-    var value = target.value;
+    var value = target.getAttribute('value');
```

Comment:
It's pretty easy to add a getter, but we've avoided doing so. I can do so if you prefer.